dot15d4/Cargo.toml

```diff
@@ -15,15 +15,19 @@ defmt = { version = "0.3", optional = true }
 bitflags = "2.4.2"
 heapless = "0.8.0"
 critical-section = "1.1"
+rand_core = { version = "0.6.4", default-features = false }
+embedded-hal-async = { version = "1.0.0" }
 
 colored = { version = "2", optional = true }
 clap = { version = "4.5.1", features = ["derive"], optional = true }
 hex = { version = "0.4.3", optional = true }
 
 [dev-dependencies]
-pollster = "0.3"
+tracing-subscriber = "0.3"
```

Comment:
Is there a reason why we still have `tracing-subscriber`?

Reply:
Yes, this should probably be removed